contracts/upgradeability/UpgradeabilityProxyFactory.sol

```diff
@@ -47,9 +47,9 @@ contract UpgradeabilityProxyFactory {
   }
 
   /**
-  * @dev Internal function to create an upgradeable proxy
-  * @return address of the new proxy created
-  */
+   * @dev Internal function to create an upgradeable proxy
+   * @return address of the new proxy created
+   */
```

Comment:
there are many indentation changes, please confirm they were intended just in case...

Reply:
yep, all checked to be homogeneous across contracts